CONTRIBUTING.md

```diff
@@ -0,0 +1,9 @@
+# Contributing
```

Comment:
I know this seems super obvious but we probably want to mention `npm install` in the contributing.md.